build/debian-hyperkube-base/Dockerfile

```diff
@@ -14,12 +14,8 @@
 
 FROM BASEIMAGE
 
-# The samba-common, cifs-utils, and nfs-common packages depend on
-# ucf, which itself depends on /bin/bash existing.
-# It doesn't seem to actually need bash, however.
-RUN ln -s /bin/sh /bin/bash
```

Comment:
I guess we could also remove the symlink after installing the packages, though that's a pretty ugly hack, too.

Reply:
Right @ixdy